### PR TITLE
adjust dependencies

### PR DESCRIPTION
**Pull request by denarch22**
## 🛠️ Changes
the dependencies are adjusted so that it installs the modules locally and stac-pgstac in its internal dependencies does not overwrite these dependencies

## 📝 Associated issues
[Usar código del repositorio para Dockerfile](https://github.com/stac-utils/stac-fastapi/compare/main...PEM-Humboldt:stac-fastapi:adjustDependencies?expand=1)

### CODE REVIEW

File: requirements.txt
Comment (erikasv) [inline requirements.txt:1]: Rename this file to `requirements.pgstac.txt`

File: stac_fastapi/api/stac_fastapi/api/app.py
Comment (erikasv) [inline stac_fastapi/api/stac_fastapi/api/app.py:1]: undo the changes in this file for this PR

File: stac_fastapi/api/stac_fastapi/api/openapi.py
Comment (erikasv) [inline stac_fastapi/api/stac_fastapi/api/openapi.py:1]: undo the changes in this file for this PR

File: stac_fastapi/api/tests/benchmarks.py
Comment (erikasv) [inline stac_fastapi/api/tests/benchmarks.py:1]: undo the changes in this file for this PR

File: stac_fastapi/api/tests/conftest.py
Comment (erikasv) [inline stac_fastapi/api/tests/conftest.py:1]: undo the changes in this file for this PR

File: stac_fastapi/api/tests/test_api.py
Comment (erikasv) [inline stac_fastapi/api/tests/test_api.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/setup.py
Comment (erikasv) [inline stac_fastapi/extensions/setup.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/stac_fastapi/extensions/core/aggregation/aggregation.py
Comment (erikasv) [inline stac_fastapi/extensions/stac_fastapi/extensions/core/aggregation/aggregation.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/stac_fastapi/extensions/core/collection_search/collection_search.py
Comment (erikasv) [inline stac_fastapi/extensions/stac_fastapi/extensions/core/collection_search/collection_search.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/stac_fastapi/extensions/core/transaction.py
Comment (erikasv) [inline stac_fastapi/extensions/stac_fastapi/extensions/core/transaction.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/stac_fastapi/extensions/third_party/bulk_transactions.py
Comment (erikasv) [inline stac_fastapi/extensions/stac_fastapi/extensions/third_party/bulk_transactions.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/tests/test_collection_search.py
Comment (erikasv) [inline stac_fastapi/extensions/tests/test_collection_search.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/tests/test_filter.py
Comment (erikasv) [inline stac_fastapi/extensions/tests/test_filter.py:1]: undo the changes in this file for this PR

File: stac_fastapi/extensions/tests/test_transaction.py
Comment (erikasv) [inline stac_fastapi/extensions/tests/test_transaction.py:1]: undo the changes in this file for this PR

File: stac_fastapi/types/stac_fastapi/types/requests.py
Comment (erikasv) [inline stac_fastapi/types/stac_fastapi/types/requests.py:1]: undo the changes in this file for this PR

File: stac_fastapi/types/stac_fastapi/types/search.py
Comment (erikasv) [inline stac_fastapi/types/stac_fastapi/types/search.py:1]: undo the changes in this file for this PR

**Review by erikasv — CHANGES_REQUESTED** [review]
In this project don't use the same format rules as we have in other projects

File: requirements.txt
Comment (denarch22) [reply requirements.txt:1]: The name was adjusted according to the comment

File: stac_fastapi/api/stac_fastapi/api/app.py [hunk 1]
```diff
@@ -410,7 +410,7 @@ def add_health_check(self):
         @mgmt_router.get("/_mgmt/ping")
         async def ping():
             """Liveliness/readiness probe."""
-            return {"message": "PONG"}
+            return {"message": "PONGO"}
```
Comment (erikasv) [inline stac_fastapi/api/stac_fastapi/api/app.py]: undo this change

Comment (denarch22) [reply stac_fastapi/api/stac_fastapi/api/app.py]: Sorry, I forgot to change it, I already did it

**Review by erikasv — CHANGES_REQUESTED** [review]
the container `stac-fastapi-pgstac` is returning an error on runtime `bash: line 1: ./scripts/wait-for-it.sh: Permission denied`

**Comment by denarch22** [comment]
> bash: line 1: ./scripts/wait-for-it.sh: Permission denied

Although the error does not happen to me, I added a command to give execution permissions on the script

**Review by erikasv — APPROVED** [review]
